### PR TITLE
feat(postcard): create postcard service

### PR DESCRIPTION
**Pull request by dmlittle**
What:
 - Create an Angular Postcard service

Why:
 - Learning!

### CODE REVIEW

File: test/services/postcard.spec.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var Config = require('../../config');
```
Comment (robinjoseph08) [inline test/services/postcard.spec.js]: nit: we usually put local `require`s after global `require`s

File: test/services/postcard.spec.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+var Angular = require('angular');
+var expect  = require('chai').expect;
```
Comment (robinjoseph08) [inline test/services/postcard.spec.js]: while we normally adhere to the alphabetical rule, whenever `expect` needs to be required, we move it to the top of the block (above `Angular`)

**Comment by robinjoseph08** [comment]
Small stuff, but LGTM 👍

File: app/services/index.js [hunk 3]
```diff
@@ -3,14 +3,16 @@
...
+var Postcard = require('./postcard');
+var Session  = require('./session');
 
 module.exports = Angular.module('postcard-create.services', [
   Cookies
 ])
 .factory('API', API)
 .factory('Auth', Auth)
 .factory('Session', Session)
+.factory('Postcard', Postcard)
```
Comment (elnaz) [inline app/services/index.js]: @robinjoseph08 should these be in alphabetical order?

Comment (robinjoseph08) [inline app/services/index.js]: Ah yes they should. Good catch

**Comment by elnaz** [comment]
LGTM pending alpha nit

**Pull request merged**